tests/unit/lcos/when_each.cpp

```diff
@@ -40,7 +40,6 @@ void test_when_each_from_list()
 
             unsigned id = fut.get();
 
-            HPX_TEST(id >= 0);
```

Comment:
this should rather be `HPX_TEST(id > 0)`, shouldn't it?

Reply:
If I understand the test correctly, no. At least `id` will be 0 the for the first future processed by `when_each` a bit further down. The important test is `HPX_TEST_EQ(id, idx)`.

Reply:
Ahh, I see now. Good.